src/Frame.cpp

```diff
@@ -922,7 +922,7 @@ std::shared_ptr<Magick::Image> Frame::GetMagickImage()
 		AddColor(width, height, "#000000");
 
 	// Get the pixels from the frame image
-	QRgb const *tmpBits = (const QRgb*)image->bits();
+	QRgb const *tmpBits = (const QRgb*)image->constBits();
```

Comment:
I'd prefer changing this one as well so it matches its cast, while we're in there:
```suggestion
	const QRgb *tmpBits = (const QRgb*)image->constBits();
```